File: 0169-majority-element/0169-majority-element.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        int rsl = 0;
        unordered_map<int, int> mp;
        for (auto it : nums) 
            mp[it]++;

        for (auto it : mp) {
            if (it.second > (int)nums.size() / 2)
                rsl = it.first;
        }

        return rsl;
    }
};

File: 0169-majority-element/0169-majority-element.py
class Solution:
    def majorityElement(self, nums: list[int]) -> int:
        num_count = Counter(nums)
        new_map = dict(sorted(num_count.items(), key=lambda item: item[1]))
        first_key = list(new_map.keys())[-1]
        return first_key